keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/diff/LogProbGradients.java

```diff
@@ -0,0 +1,51 @@
+package io.improbable.keanu.vertices.dbl.nonprobabilistic.diff;
+
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.vertices.VertexId;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LogProbGradients {
```

Comment:
Can we add javadoc to this? Is `LogProbGradients` now managing the summed partials which was previously being done by each `PartialDerivatives`?

Reply:
👍